### PR TITLE
feat: add linea

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/hooks/useKyberSwapConfig.ts [hunk 1]
```diff
@@ -65,7 +65,8 @@ const parseGlobalResponse = (
   chainId: ChainId,
 ): KyberswapGlobalConfig => {
   const data = responseData?.data?.config
-  const aggregatorDomain = data?.aggregator ?? DEFAULT_AGGREGATOR_API
+  const aggregatorDomain =
+    chainId === ChainId.LINEA_TESTNET ? DEFAULT_AGGREGATOR_API : data?.aggregator ?? DEFAULT_AGGREGATOR_API
```
Comment (namgold) [inline src/hooks/useKyberSwapConfig.ts:69]: Why would we need this

File: src/pages/About/AboutKyberSwap/index.tsx [hunk 2]
```diff
@@ -168,7 +169,7 @@ export const KSStatistic = () => {
         <ForTraderInfoRow>
           <ForTraderInfoCell>
             <Text fontWeight="600" fontSize="24px">
-              {MAINNET_NETWORKS.length - 1}+
+              {MAINNET_NETWORKS.length}+
```
Comment (namgold) [inline src/pages/About/AboutKyberSwap/index.tsx:172]: Why

Comment (viet-nv) [reply src/pages/About/AboutKyberSwap/index.tsx:172]: why length -1?

File: src/hooks/useKyberSwapConfig.ts (hunk 1 above)
Comment (viet-nv) [reply src/hooks/useKyberSwapConfig.ts:69]: Meta prod needs authorization. then for linea testnet, just directly call aggregator is better

File: src/components/Header/groups/SwapNavGroup.tsx [hunk 3]
```diff
@@ -124,7 +125,7 @@ const SwapNavGroup = () => {
             </StyledNavLink>
           )}
 
-          {isSolana || (
+          {(isSolana || chainId !== ChainId.LINEA_TESTNET) && (
```
Comment (nguyenhoaidanh) [inline src/components/Header/groups/SwapNavGroup.tsx:128]: wrong condition, should be
!isSolana && chainId !== ChainId.LINEA_TESTNET

File: src/hooks/useKyberSwapConfig.ts (hunk 1 above)
Comment (namgold) [reply src/hooks/useKyberSwapConfig.ts:69]: We might need to make `aggregator` config chain based

File: src/pages/About/AboutKyberSwap/index.tsx (hunk 2 above)
Comment (namgold) [reply src/pages/About/AboutKyberSwap/index.tsx:172]: Because it's correct. If we support 14 chain, we can show that it's 13+ :trollface: not 14+

Comment (viet-nv) [reply src/pages/About/AboutKyberSwap/index.tsx:172]: 14+ mean more is coming

File: src/components/Header/groups/SwapNavGroup.tsx (hunk 3 above)
Comment (viet-nv) [reply src/components/Header/groups/SwapNavGroup.tsx]: nice catch, I've updated

File: .env.adpr [hunk 4]
```diff
@@ -1,45 +1,44 @@
-VITE_DEFAULT_AGGREGATOR_API=https://meta-aggregator.dev.kyberengineering.io
-VITE_AGGREGATOR_API=https://aggregator-api.dev.kyberengineering.io
-VITE_AGGREGATOR_STATS_API=https://aggregator-stats.dev.kyberengineering.io
+VITE_DEFAULT_AGGREGATOR_API=https://aggregator-api.kyberswap.com
```
Comment (viet-nv) [inline .env.adpr:1]: TODO: revert env

File: src/hooks/useKyberSwapConfig.ts (hunk 1 above)
Comment (nguyenhoaidanh) [reply src/hooks/useKyberSwapConfig.ts:69]: DEFAULT_AGGREGATOR_API is meta

rename DEFAULT_AGGREGATOR_API => META_AGGREGATOR_API to prevent confused

Comment (viet-nv) [reply src/hooks/useKyberSwapConfig.ts:69]: will revert now. this is only for testing

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**